django-stubs/views/decorators/vary.pyi

```diff
@@ -1,7 +1,8 @@
+import functools
```

Comment:
Hehe this one isn't needed either

```suggestion
```

Reply:
Extremely sorry! That one just slipped through. Thanks a ton for the help.